src/main/java/com/wojtek/parkingmeter/services/TicketServiceImpl.java

```diff
@@ -0,0 +1,140 @@
+package com.wojtek.parkingmeter.services;
+
+import com.wojtek.parkingmeter.helpers.*;
+import com.wojtek.parkingmeter.helpers.enums.HasStartedEnum;
+import com.wojtek.parkingmeter.helpers.enums.TicketType;
+import com.wojtek.parkingmeter.mapper.TicketMapper;
+import com.wojtek.parkingmeter.model.CarEntity;
+import com.wojtek.parkingmeter.model.TicketEntity;
+
+import com.wojtek.parkingmeter.model.TicketDTO;
+import com.wojtek.parkingmeter.repositories.CarRepository;
+import com.wojtek.parkingmeter.repositories.TicketRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+@Service
+public class TicketServiceImpl implements TicketService {
+
+    private final TicketRepository ticketRepository;
+    private final CarRepository carRepository;
+    private final TicketMapper ticketMapper;
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public TicketServiceImpl(TicketRepository ticketRepository, CarRepository carRepository, TicketMapper ticketMapper) {
+        this.ticketRepository = ticketRepository;
+        this.carRepository = carRepository;
+        this.ticketMapper = ticketMapper;
+    }
+
+    @Override
+    public TicketDTO startTicket(String ticket_type, String nr_plate) {
+
+        TicketEntity newTicketEntity = new TicketEntity(TicketType.valueOf(ticket_type.toUpperCase()), LocalDateTime.now(), LocalDateTime.of(0, 1, 1, 0, 0, 0, 0));
```

Comment:
czy używanie `now()` będzie dobre jeśli będziemy to chcieli dobrze przetestować?

Reply:
Co sugerujesz w zamian? Nie wiem jak zrobić lepiej/inaczej.
W sumie miałem kłopot z przetestowaniem. Mianowicie nie mogłem dać żadnej wartości oczekiwanej bo startStamp zawsze się będzie różniło.